lib/src/application/auth_bloc/auth_state.dart

```diff
@@ -13,7 +13,7 @@ class AuthState with _$AuthState {
     required Status sendOtpStatus,
     
     
-    required bool isVrifyingOtp,
+    required Status isVrifyingOtp,
```

Comment:
change the name of variable `isVerifyingOtp` to more readable and relatable to status like `otpVerificationStatus`